src/wallet.rs

```diff
@@ -491,7 +490,7 @@ impl<F: SecureWalletFile + Debug> Wallet<F> {
                 &mut rng,
                 index,
                 index,
-                refund_addr.psk(),
+                addr.psk(),
```

Comment:
like it's done in the unstake process right? that's the same principal

Reply:
indeed